fixtures/dom/src/components/fixtures/textareas/index.js

```diff
@@ -39,6 +40,43 @@ export default class TextAreaFixtures extends React.Component {
             <textarea placeholder="Hello, world" />
           </div>
         </TestCase>
+
+        <TestCase
+          title="Required Textareas"
+          affectedBrowsers="Firefox"
+          relatedIssues="16402">
+          <TestCase.Steps>
+            <li>View this test in Firefox</li>
+          </TestCase.Steps>
+
+          <TestCase.ExpectedResult>
+            You should{' '}
+            <b>
+              <i>not</i>
+            </b>{' '}
+            see a red aura, indicating the textarea is invalid.
```

Comment:
Can you explain this a bit better? I _do_ see a red outline specifically when I focus the "Empty value prop string" textarea, hit any character on the keyboard, and then move the focus to one of the other textareas. Is this expected?

Reply:
Ah that seems reasonable. Maybe update the instructions then to be clearer about this case. 

> I think these are setup this way to avoid needing to scaffold out the controlled input code. Would it be less confusing if they were set up this way?

Not quite sure what you mean by that but I think what we have now is already better than what we had 🙂

Reply:
updated expected result to read "...see a red aura **_on initial page load_**, indicating the textarea is invalid..."

do you think this is sufficient clarification?